### PR TITLE
Streets GL link

### PR DESCRIPTION
**Pull request by igor-suhorukov**
Streets GL - OpenStreetMap 3D renderer powered by WebGL2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,7 @@ This section is a great place to start if you want to get into improving OpenStr
 * [indoor=](https://indoorequal.org) - Map for viewing indoor data from OpenStreetMap. ([Wiki](https://wiki.openstreetmap.org/wiki/Indoor%3D)) <!-- markdown-link-check-disable-line -->
 * [Qwant Maps](https://www.qwant.com/maps) - Modern map with POI search and routing functionality. ([Wiki](https://wiki.openstreetmap.org/wiki/Qwant_Maps))
 * [F4map Demo](https://demo.f4map.com/) - 3D rendering demo of OpenStreetMap data. ([Wiki](https://wiki.openstreetmap.org/wiki/F4_Map))
+* [Streets GL](https://github.com/StrandedKitty/streets-gl) OpenStreetMap 3D renderer powered by WebGL2 ([Wiki](https://wiki.openstreetmap.org/wiki/Streets_GL))
```
Comment (Unkn0wnCat) [inline README.md:156]: You've inserted this line in the middle. As per Awesome List guidelines you should always add to the bottom of the category.

Please move this to the bottom of "Web Maps"

**Review by Unkn0wnCat — CHANGES_REQUESTED** [review]
Thanks for this addition! It's a really awesome project and I'd love to have it on our list, but please perform the attached change.

After that I'll be happy to merge 💯

File: README.md (hunk 1 above)
Comment (igor-suhorukov) [reply README.md]: Sure. Thank you for review!!!
DONE

**Review by Unkn0wnCat — APPROVED** [review]

**Comment by Unkn0wnCat** [comment]
Thanks for the addition and for the quick change 😛

It's an awesome project which will be right at home here 🎉

**Pull request merged**